mne/datasets/limo/limo.py

```diff
@@ -242,3 +236,9 @@ def load_data(subject, path=None, force_update=False, update_path=None,
     epochs.info['bads'] = missing_chans  # missing channels are marked as bad.
 
     return epochs
+
+###############################################################################
+# References
+# ----------
+#
```

Comment:
remove the blank line here.  Otherwise our style checker will complain that "docstring sections should not start with a blank line"

Reply:
also, shouldn't the `.. footbibliography::` go rather to line 91, and the currently deleted "References" be restored?

```
References
----------
.. footbibliography::
```
?

Reply:
ah, yes.  The `References` section should remain in the docstring.  It only goes at the end of the file in examples/tutorials.

Reply:
(so nice to have @sappelhoff around to catch these errors that I keep missing!)